CHANGELOG.md

```diff
@@ -60,6 +60,9 @@ The format is based on [Keep a Changelog].
     operations (+,-,*,/). (#2537)
 -   A `ParmeterVector` class has been added to ease the construction of circuits
     requiring a large number of parameters. (#2379)
+-   The option `with_layout` was added to the circuit drawer
+    method. This option allows to include layout information if its available in
+    the circuit to draw (\#2739)
```

Comment:
We might want to reword this slightly since you've updated the default. But it's not a blocker. We'll cover those details in the release notes.

Reply:
Updated (and moved to the section `changed`) in 56853a5c